### PR TITLE
 homework after messy commites

### PR DESCRIPTION
**Pull request by islam-fawzy25**
sorry for mess 
please try to open each file and check it

### CODE REVIEW

File: nodejs/week1/homework/warmup.js [hunk 1]
```diff
@@ -1,35 +1,36 @@
...
+
 class circle {
     constructor(radius){
         this.raduis=radius
     }
-
-    getDiameter(){
-        
+    getDiameter(){    
       return  console.log(2 * this.raduis);
```
Comment (florinmaticu) [inline nodejs/week1/homework/warmup.js:9]: Returning the output of `console.log` is not really useful here. For this function, it is expected to return a number.
```suggestion
    const diameter = 2 * this.radius
    console.log(diameter)
     return diameter  ;
```

File: nodejs/week1/homework/warmup.js [hunk 2]
```diff
@@ -1,35 +1,36 @@
 console.log("inside warmup file");
 
+
 class circle {
     constructor(radius){
         this.raduis=radius
```
Comment (florinmaticu) [inline nodejs/week1/homework/warmup.js:6]: small typo
```suggestion
        this.radius=radius
```

File: nodejs/week1/homework/warmup.js [hunk 3]
```diff
@@ -1,35 +1,36 @@
...
     }
-
-    getDiameter(){
-        
+    getDiameter(){    
       return  console.log(2 * this.raduis);
     }
-
     getCircumference(){
         return console.log(2 * 3.14 * this.raduis);
```
Comment (florinmaticu) [inline nodejs/week1/homework/warmup.js:12]: see the comment from the function with the diameter.

Comment (islam-fawzy25) [reply nodejs/week1/homework/warmup.js]: Thanks it was really helpful

**Pull request merged**